src/templates/party-template.js

```diff
@@ -362,10 +461,67 @@ const PartyPage = props => {
         </div>
       </section>
 
+      <section
+        css={{
+          ...cssSection,
+          ...cssStickyContainer,
+          background: "var(--cl-white)",
+        }}
+      >
+        <div
+          ref={navRef}
+          css={{
+            display: "flex",
+            justifyContent: "space-between",
+          }}
+        >
+          <div ref={voteLogsMenuRef} css={{ ...cssStickyMenu }}>
```

Comment:
The element that can be clicked should be either
- `<a>` if clicking it just change the URL (In this case, if we use a link anchor to smooth scroll)
- `<button>` for other cases eg. call some JavaScript function or submit a form

This improves the accessibility, for SEO and screen readers for blind people. so they will know this element can be clicked. (And we also don't need `pointer: curser` in the CSS cause browser will do that for us).